ios/FluentUI.Demo/FluentUI.Demo/Demos/SearchBarDemoController.swift

```diff
@@ -7,6 +7,22 @@ import FluentUI
 import UIKit
 
 class SearchBarDemoController: DemoController, SearchBarDelegate {
+    private lazy var searchBarWithBadgeView: SearchBar = {
+        buildSearchBar(
+            autocorrectionType: .yes,
+            placeholderText: "Type badge to add a badge"
```

Comment:
nit: can you put the first badge word between quotation marks to make it easier to understand?

`Type \"badge\" to add a leading badge`

Reply:
Done